src/anemoi/training/diagnostics/plots.py

```diff
@@ -433,42 +433,90 @@ def plot_flat_sample(
             norm=TwoSlopeNorm(vcenter=0.0),
             title=f"{vname} pred err",
         )
-    else:
-        scatter_plot(fig, ax[1], lon=lon, lat=lat, data=truth, title=f"{vname} target")
-        scatter_plot(fig, ax[2], lon=lon, lat=lat, data=pred, title=f"{vname} pred")
+    elif vname == "mwd":
+        cyclic_colormap = "twilight"
+
+        def error_plot_in_degrees(array1: np.ndarray, array2: np.ndarray) -> np.ndarray:
+            """Calculate error between two arrays in degrees in range [-180, 180]."""
+            tmp = (array1 - array2) % 360
+            return np.where(tmp > 180, tmp - 360, tmp)
+
+        sample_shape = truth.shape
+        pred = np.maximum(np.zeros(sample_shape), np.minimum(360 * np.ones(sample_shape), (pred)))
+        scatter_plot(fig, ax[1], lon=lon, lat=lat, data=truth, cmap=cyclic_colormap, title=f"{vname} target")
+        scatter_plot(fig, ax[2], lon=lon, lat=lat, data=pred, cmap=cyclic_colormap, title=f"capped {vname} pred")
+        err_plot = error_plot_in_degrees(truth, pred)
         scatter_plot(
             fig,
             ax[3],
             lon=lon,
             lat=lat,
-            data=truth - pred,
+            data=err_plot,
             cmap="bwr",
             norm=TwoSlopeNorm(vcenter=0.0),
-            title=f"{vname} pred err",
-        )
-
-    if sum(input_) != 0:
-        scatter_plot(fig, ax[0], lon=lon, lat=lat, data=input_, title=f"{vname} input")
-        scatter_plot(
-            fig,
-            ax[4],
-            lon=lon,
-            lat=lat,
-            data=pred - input_,
-            cmap="bwr",
-            norm=TwoSlopeNorm(vcenter=0.0),
-            title=f"{vname} increment [pred - input]",
+            title=f"{vname} pred err: {np.nanmean(np.abs(err_plot)):.{4}f} deg.",
         )
+    else:
+        scatter_plot(fig, ax[1], lon=lon, lat=lat, data=truth, title=f"{vname} target")
+        scatter_plot(fig, ax[2], lon=lon, lat=lat, data=pred, title=f"{vname} pred")
         scatter_plot(
             fig,
-            ax[5],
+            ax[3],
             lon=lon,
             lat=lat,
-            data=truth - input_,
+            data=truth - pred,
             cmap="bwr",
             norm=TwoSlopeNorm(vcenter=0.0),
-            title=f"{vname} persist err",
+            title=f"{vname} pred err",
         )
+
+    if sum(input_) != 0:
+        if vname == "mwd":
```

Comment:
we should put this on a list to revisit when we refactor the plotting

Reply:
maybe a config setting in the future, but ok for now I think

Reply:
I added an issue under "later"